src/components/NavBar.js

```diff
@@ -1,13 +1,22 @@
+import { ImUser } from 'react-icons/im';
 import { Link } from 'react-router-dom';
+import styles from './NavBar.module.css';
 
 const NavBar = () => (
   <div className="navbar">
-    <header className="header">
-      <h1 className="navbar-title">Book Store</h1>
-      <nav className="nav">
-        <Link className="nav-link" to="/">Books</Link>
-        <Link className="nav-link" to="/Categories">Categories</Link>
-      </nav>
+    <header className={styles.header}>
+      <div className={styles.wrapper}>
+        <h1 className={styles.heading_text}>Bookstore CMS</h1>
+        <nav className={styles.navigation}>
+          <ul>
+            <li><Link className="nav-link" to="/">Books</Link></li>
+            <li><Link className="nav-link" to="/Categories">Categories</Link></li>
+          </ul>
+        </nav>
```

Comment:
- [ ] As for this project 👍 your design should be exaactly as presented in the [Zephin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) , Kindly make sure the Logo and the navlinks are well aligned 👍 

- [ ] Kindly uppercase the navlinks text 👍 to stick to the figma design

### Your design

![Capture](https://github.com/binyamolango/book-store/assets/80458199/3b21f46b-e63a-400c-b293-273c1a3b2e89)

### Zephin design

![Capture](https://github.com/binyamolango/book-store/assets/80458199/7e578358-303d-4aeb-8c78-a5476755db9f)